CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/gui/MultiChoiceInputPane.java

```diff
@@ -125,6 +128,15 @@ public MultiChoiceInputPane(final PluginParameter<MultiChoiceParameterValue> par
             });
         });
         getChildren().add(field);
+        parameterId = parameter.getId();
+        List<String> multiChoiceRecentValues = RecentParameterValues.getRecentValues(parameterId);
+        if (multiChoiceRecentValues != null) {
+            if (multiChoiceRecentValues.size() > 1) {
+                parameter.setStringValue(multiChoiceRecentValues.get(1));
+            } else {
+                parameter.setStringValue(multiChoiceRecentValues.get(0));
+            }
```

Comment:
if-else can be converted to tertiary (similar to suggestion for `ColorInputPane`)